src/test/unit/algorithms/TemporalMemoryTest.cpp

```diff
@@ -499,10 +499,10 @@ TEST(TemporalMemoryTest, NoNewSegmentIfNotEnoughWinnerCells) {
       /*predictedSegmentDecrement*/ 0.0,
       /*seed*/ 42);
 
-  const UInt zeroColumns[0] = {};
+  const UInt zeroColumns[1] = {0}; // MSVC requires at least one element.
   const UInt activeColumns[1] = {0};
 
-  tm.compute(0, zeroColumns);
+  tm.compute(0, zeroColumns);  // The actual size of the array is not relevent.
```

Comment:
if it really is the same, zeroColumns test and activeCols are now same, remove one.